Whisky/Utils/Logger.swift

```diff
@@ -0,0 +1,68 @@
+//
+//  Logger.swift
+//  Whisky
+//
+//  Created by Isaac Marovitz on 08/06/2023.
+//
+
+import Foundation
+import OSLog
+
+class Log {
+    static let logsFolder = FileManager.default.urls(for: .libraryDirectory,
+                                                    in: .userDomainMask)[0]
+        .appendingPathComponent("Logs")
+        .appendingPathComponent("Whisky")
+
+    let fileHandle: FileHandle
+    let logger: Logger
+
+    init?() {
+        do {
+            if !FileManager.default.fileExists(atPath: Log.logsFolder.path) {
+                try FileManager.default.createDirectory(at: Log.logsFolder, withIntermediateDirectories: true)
+            }
+
+            let formatter = DateFormatter()
+            formatter.dateFormat = "d MMM y HH-mm"
```

Comment:
Can you create the `DateFormatter` somewhere it's less likely to be recreated?
It's usually an expensive instance to create, but only for the first usage of each instance.